src/install/install.zig

```diff
@@ -8410,12 +8410,14 @@ pub const PackageManager = struct {
         ctx: Command.Context,
         comptime log_level: PackageManager.Options.LogLevel,
     ) !PackageInstall.Summary {
-        var lockfile = this.lockfile;
+        const original_lockfile = this.lockfile;
+        defer this.lockfile = original_lockfile;
```

Comment:
is this needed? We want `PackageManager` to have the lockfile before `clean()` for printing the summary?

Reply:
The lockfile stored on `this` must be the expected instance, or else either git dependencies could fail or it will attempt to install dev dependencies